### DISCUSSION TITLE
Lesson 7 (testing one test at a time)

### DISCUSSION THREAD

**Discussion by Abhishek-2416** [Cyfrin/foundry-full-course-cu#975] [Q&A] [answered]
When we have to just test a single function we need to write
forge test -m functionName -vv --fork-url Sepolia 
but it throws me an error 
error: unexpected argument '-m' found

Can someone tell whats wrong i tried reading through forge test --help too didnt help much xd

**Comment by sunnyStefi** [comment, marked as answer]
-m is deprecated, please use --mt

**Reply by Abhishek-2416** [reply]
Great Thankyou
